src/main/java/com/onevizion/scmdb/DbManager.java

```diff
@@ -26,6 +27,7 @@
     private static final String NO_SCRIPTS_TO_EXEC_MSG = "No scripts to execute in [{}]:";
     private static final String SCRIPTS_TO_EXEC_MSG = "\nScripts to be executed in [{}]:";
     private static final String ROLLBACKS_TO_SKIP_MSG = "\nRollbacks skipped in [{}]:";
+    private static final String SCRIPT_NUMBERING_IS_MORE_THAN_THREE_DIGITS_REGEX = "^\\d{4,}_.*";
```

Comment:
if constant name is correct than you have implemented `Run dev DB scripts with number less than 1000 at the end` not lestt than 100,fix it

Reply:
There is a misprint in regx, so the name is not correct. Fixed.